### PR TITLE
[Analyzer] update analyzer and use it in test contracts

### PR DESCRIPTION
**Pull request by Jim8y**
This pr update the neo contract syntax analyzer and use it in the test contracts.

### CODE REVIEW

File: tests/Neo.Compiler.CSharp.TestContracts/Contract_TryCatch.cs [hunk 1]
```diff
@@ -7,8 +7,10 @@ public class Contract_TryCatch : SmartContract.Framework.SmartContract
     {
         [ByteArray("0a0b0c0d0E0F")]
         private static readonly ByteString invalidECpoint = default!;
+
         [ByteArray("024700db2e90d9f02c4f9fc862abaca92725f95b4fddcc8d7ffa538693ecf463a9")]
-        private static readonly ByteString byteString2Ecpoint = default!;
+        private static readonly ByteString byteString2Ecpoint =
+            "024700db2e90d9f02c4f9fc862abaca92725f95b4fddcc8d7ffa538693ecf463a9";
```
Comment (Jim8y) [inline tests/Neo.Compiler.CSharp.TestContracts/Contract_TryCatch.cs:13]:
```suggestion
        private static readonly ByteString byteString2Ecpoint = default!;
```

File: tests/Neo.Compiler.CSharp.TestContracts/Contract_TryCatch.cs [hunk 2]
```diff
@@ -7,6 +7,7 @@ public class Contract_TryCatch : SmartContract.Framework.SmartContract
     {
         [ByteArray("0a0b0c0d0E0F")]
         private static readonly ByteString invalidECpoint = default!;
+
```
Comment (Jim8y) [inline tests/Neo.Compiler.CSharp.TestContracts/Contract_TryCatch.cs:10]:
```suggestion
```

**Review by shargon — DISMISSED** [review]

**Comment by Jim8y** [comment]
Will update this after i merge other prs, since will update supported systemcalls.

**Comment by Jim8y** [comment]
Finally~~~~~!!!!!!

**Review by shargon — APPROVED** [review]

**Pull request merged**